src/overlays/actors/ovl_Item_B_Heart/z_item_b_heart.c

```diff
@@ -10,26 +16,103 @@ void ItemBHeart_Destroy(Actor* thisx, PlayState* play);
 void ItemBHeart_Update(Actor* thisx, PlayState* play);
 void ItemBHeart_Draw(Actor* thisx, PlayState* play);
 
-/*
-const ActorInit Item_B_Heart_InitVars = {
-    ACTOR_ITEM_B_HEART,
-    ACTORCAT_BOSS,
-    FLAGS,
-    OBJECT_GI_HEARTS,
-    sizeof(ItemBHeart),
-    (ActorFunc)ItemBHeart_Init,
-    (ActorFunc)ItemBHeart_Destroy,
-    (ActorFunc)ItemBHeart_Update,
-    (ActorFunc)ItemBHeart_Draw
+void ItemBHeart_UpdateRotationAndScale(ItemBHeart* this, PlayState* play);
+
+const ActorInit Item_B_Heart_InitVars = { ACTOR_ITEM_B_HEART,
+                                          ACTORCAT_BOSS,
+                                          FLAGS,
+                                          OBJECT_GI_HEARTS,
+                                          sizeof(ItemBHeart),
+                                          (ActorFunc)ItemBHeart_Init,
+                                          (ActorFunc)ItemBHeart_Destroy,
+                                          (ActorFunc)ItemBHeart_Update,
+                                          (ActorFunc)ItemBHeart_Draw };
```

Comment:
Could you add a trailing comma and reformat?
```suggestion
const ActorInit Item_B_Heart_InitVars = { ACTOR_ITEM_B_HEART,
                                          ACTORCAT_BOSS,
                                          FLAGS,
                                          OBJECT_GI_HEARTS,
                                          sizeof(ItemBHeart),
                                          (ActorFunc)ItemBHeart_Init,
                                          (ActorFunc)ItemBHeart_Destroy,
                                          (ActorFunc)ItemBHeart_Update,
                                          (ActorFunc)ItemBHeart_Draw, };
```